.github/workflows/ci_test.yml

```diff
@@ -94,3 +94,47 @@ jobs:
 #      with:
 #          name: documentation
 #          path: .tox/docs_out/
+  ci-tests-macos:
+    runs-on: macos-latest
+    steps:
+        - uses: actions/checkout@v4
+        - name: Set-Up Python Env
+          uses: mamba-org/setup-micromamba@v1
+          with: 
+            init-shell: bash
+            environment-name: pint
+            cache-environment: true
+            cache-downloads: true
+            create-args: >-
+              --platform osx-64
+              -c conda-forge
+              python=3.11
```

Comment:
Is there a reason why this is not Python 3.13?

Reply:
no, just trying to distinguish from the tox version.  Now it's 3.13.